golang/cosmos/x/vstorage/keeper/keeper.go

```diff
@@ -119,13 +119,19 @@ func NewKeeper(storeKey storetypes.StoreKey) Keeper {
 	}
 }
 
-var MetricKeysSizeDelta = []string{"store", "size_delta"}
+var MetricKeyStoreAllocated = []string{"store", "allocated"}
+var MetricKeyStoreReleased = []string{"store", "released"}
 const MetricLabelStoreKey = "storeKey"
 
-func NewMetricsLabels(k Keeper) []metrics.Label {
-	return []metrics.Label{
+func ReportStoreSizeMetrics(k Keeper, sizeDelta float32) {
```

Comment:
Why not make this a method on the Keeper?
```suggestion
// ReportStoreSizeMetrics exports sizeDelta metrics when
// Cosmos telemetry is enabled.
func (k Keeper) ReportStoreSizeMetrics(sizeDelta float32) {
```

Reply:
done.